diff --git a/10_splash_damage.lua b/10_splash_damage.lua
index 79a6e33..d274515 100644
--- a/10_splash_damage.lua
+++ b/10_splash_damage.lua
@@ -264,6 +264,20 @@ end
 
 function onWpnEvent(event)
   if event.id == world.event.S_EVENT_SHOT then
+    --[[
+    What follows is a work-around for what is presumed to be a DCS-bug:
+    https://forum.dcs.world/topic/353679-possible-bug-in-objectgettypename-during-s_event_kill/
+    ]]--
+    if event.weapon then
+      local status, retval = pcall(event.weapon.getTypeName, event.weapon)
+      if not status then
+        -- gameMsg(tostring(event.id)..'  WTF?\n'..tostring(retval))
+        -- gameMsg(mist.utils.tableShow(event))
+        -- gameMsg(tostring(event.weapon.getTypeName))
+        return
+      end
+    end
+    -- end of work-around
     if event.weapon then
       local ordnance = event.weapon
       local weapon_desc = ordnance:getDesc()
